thirdeye/thirdeye-frontend/app/styles/components/te-navbar.scss

```diff
@@ -57,6 +57,33 @@
     }
   }
 
+  &__help-icon {
+    font-size: 24px;
+  }
+
+  &__dropdown-link {
+    display: flex; 
+    align-items: center;
+
+    &:hover .te-icon {
+      color: white;
+    }
+  }
+
+  &__dropdown-wrapper {
+    position: absolute;
+    text-align: left;
+    background-color: white;
+    z-index: 10;
+    right: 0;
+    top: 50px;
+    padding: 8px 0;
+    border-bottom-left-radius: 5px;
+    border-bottom-right-radius: 5px;
+    border: 1px solid rgba(0,0,0,.15);
```

Comment:
if opacity can be less granular, then we can use `app-shade(black, 1)` in these cases... otherwise, just keep it this way.